src/shared/components/ShareWithColleagues/ShareWithColleagues.scss

```diff
@@ -19,30 +19,36 @@ $borderLight: 1px solid $color-gray-100;
 		color: $color-error;
 	}
 
-	&__select.c-button.c-button--primary {
-		width: 100%;
-		border: $borderLight;
-		border-right: none;
-		background-color: $g-base-background;
-		text-transform: capitalize;
+	&__select.c-button {
+		.c-button__content {
+			justify-content: space-between;
+		}
 
-		&:hover,
-		:active,
-		:active:focus {
+		&.c-button--primary {
```

Comment:
we should also fix the hover issue:
![image](https://github.com/viaacode/avo2-client/assets/1710840/7d3d47f4-597a-445b-a22e-62796c8f7638)

![image](https://github.com/viaacode/avo2-client/assets/1710840/440cb0bc-692b-473e-98e5-8df1b27238b8)

The correct way of doing this is to use Button type="secondary" or maybe tertiary
and not override the styling of the primary button style